### DISCUSSION TITLE
RabbitMQ的理解 | 云泽的博客

### DISCUSSION THREAD

**Discussion by giscus** [codeyunze/blog#4] [General]
# RabbitMQ的理解 | 云泽的博客

RabbitMQ的理解 基础理解 Connections连接 表示客户端与RabbitMQ之间的TCP连接； Channels信道 客户端与RabbitMQ建立了连接之后，会分配一个AMQP信道Channel； 一个通信的通道； 一个Connection连接可以有多个Channel（便于客户端进行多线程连接）； Exchanges交换机 主要功能是用于...

http://localhost:8080/blog/mq/RabbitMQ%E7%9A%84%E7%90%86%E8%A7%A3.html

**Comment by codeyunze** [comment]
1234567890

**Comment by codeyunze** [comment]
test
